doc/pveclibmaindox.h

```diff
@@ -414,6 +416,93 @@
 *  provides inspiration for SIMD versions of; count leading zeros,
 *  population count, parity, etc.
 *
+*  \subsubsection mainpage_endian_issues_1_1 General Endian Issues
+*
+*  For POWER8, IBM made the explicit decision to support Little Endian
+*  (<B>LE</B>) data format in the Linux ecosystem.  The goal was to
+*  enhance application code portability across Linux platforms.
+*  This goal was integrated into the OpenPOWER ELF V2
+*  Application Binary Interface <B>ABI</B> specification.
+*
+*  The POWER8 processor architecturally supports an <I>Endian Mode</I>
+*  and supports both BE and LE storage access in hardware.
+*  However register to register operations do not change
+*  (are not effected by endian mode).  The ABI extends the LE
+*  storage format to vector register (logical)
+*  element numbering.
```

Comment:
Should this be explained more?  I.e. what is "vector register (logical) element numbering"?

Reply:
Best to reference the ABI here

\sa http://openpowerfoundation.org/wp-content/uploads/resources/leabi/content/dbdoclet.50655244_pgfId-1095944.html